blog/2024-04-22-feature-observability-semantic-conventions.md

```diff
@@ -0,0 +1,46 @@
+---
+slug: "feature-observability-semantic-conventions"
+title: "Feature Observability Semantic Conventions"
+date: 2024-04-22
+authors: [dyladan]
+description: "Help us make feature observability better for everyone!"
+tags: [otel, spec, specification, opentelemetry, observability]
+draft: false
+---
+
+As comic book aficionados say, with great power comes great responsibility, or great complexity as the case may be.
```

Comment:
Please add a sentence that ties this back to the challenges seen in feature flagging. It's done in the second paragraph but that's "below the fold".